bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/auth/DefaultAuthentication.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.library.auth;
```

Comment:
I'd suggest to move this class to org.eclipse.smarthome.core.auth as well and remove the library package.